cmd/soci-snapshotter-grpc/main.go

```diff
@@ -203,6 +206,15 @@ func main() {
 	log.G(ctx).Info("Exiting")
 }
 
+// Copied from https://github.com/containerd/containerd/blob/041743e8afd2bb2189aa904f04bf87b9073892d3/cmd/containerd/server/namespace.go#L34
```

Comment:
If this was copied, then we need to include containerd's copyright somewhere.

Reply:
Done